CTP/ha_repl/src/com/navercorp/cubridqa/ha_repl/Main.java

```diff
@@ -69,7 +69,7 @@ public static void exec(String configFilename) throws Exception {
 			throw new Exception("Not found test cases directory. Please check 'scenario' in test configuration file.");
 
 		String cubridPackageUrl = context.getCubridPackageUrl();
-		if (cubridPackageUrl != null && cubridPackageUrl.trim().length() > 0) {
+		if (com.navercorp.cubridqa.common.CommonUtils.isEmpty(cubridPackageUrl)) {
```

Comment:
Please fix the changes:
-		if (cubridPackageUrl != null && cubridPackageUrl.trim().length() > 0) {
+		if (com.navercorp.cubridqa.common.CommonUtils.isEmpty(cubridPackageUrl)) {